test/scenarios-periodics/el93-crel@upgrade-from-el92-prel.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Scenario tests upgrade from RHEL9.2 to RHEL9.3 (including MicroShift).
+
+start_commit="rhel-9.2-microshift-4.$(previous_minor_version)"
+target_commit=rhel-9.3-microshift-crel
+
+scenario_create_vms() {
+    if ! does_commit_exist "${target_commit}"; then
+        echo "Commit '${target_commit}' not found in ostree repo - skipping test"
+        return 0
+    fi
+    prepare_kickstart host1 kickstart.ks.template "${start_commit}"
+    launch_vm host1
+}
+
+scenario_remove_vms() {
+    if ! does_commit_exist "${target_commit}"; then
```

Comment:
Cleanup should be safe even if the VM was not created, so I don't think this is needed.